setup.py

```diff
@@ -24,7 +24,7 @@ def find_stubs(package):
 
 install_requires = [
     'six',
-    'botocore>=1.11.0',
+    'botocore>=1.12.167',
```

Comment:
Can we just set this to the earliest version we need (`1.12.54`)? Requiring the latest version is more likely to introduce conflicts with other packages

Reply:
Done.